assets/js/blocks/cart-checkout/cart/full-cart/checkout-button/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import Button from '@woocommerce/base-components/button';
+import { CHECKOUT_URL } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import { AmericanExpressLogo } from './american-express-logo';
+import './style.scss';
+import { MastercardLogo, VisaLogo } from './payment-logos'; // @todo we want to import this from `@automattic/composite-checkout` when it's published in NPM
+
+/**
+ * Checkout button rendered in the full cart page.
+ */
+const CheckoutButton = () => {
+	return (
+		<Fragment>
+			<Button
+				className="wc-block-cart__submit-button"
+				href={ CHECKOUT_URL }
```

Comment:
I think this should be provided as a prop and it's populated via whatever is consuming this component.  Otherwise we're tightly coupling global settings to this component which makes it more difficult to export should we do so down the road.

Reply:
This component is expected to be used only in the Cart component. It uses the `<Button>` from `/base/components`, which is the generic button component that can be reused in other places where we need a button. Maybe I should change the name of `Button` so we don't confuse it with the Gutenberg component?

Reply:
Okay, let's leave it for now.  No need to change the name, it should be clear from the import source where it comes from.